docs/01-marketing/01-stakeholders.md

```diff
@@ -0,0 +1,7 @@
+# Stakeholders List
+
+| Name                  | Role                 | Category   | Influence | Interest | Interests                              | Contacts                  | Contact frequency | Etc                                                       |
+|-----------------------|----------------------|------------|-----------|----------|----------------------------------------|---------------------------|-------------------|-----------------------------------------------------------|
+| Comunity              | client               | external   | high      | high     | terms, functions                       | github.com                | every month       |                                                           |
```

Comment:
There is a couple of types of clients
1. A person deciding to use our service in his/her product/infrastructure.
2. A person who uses our service for commenting
3. The owner of content that is commented by person in item 2.
They all should be considered as stakeholders. And their interests should be taken into account.